scripts/azureml-assets/azureml/assets/update_assets.py

```diff
@@ -63,6 +65,35 @@ def release_tag_exists(asset_config: assets.AssetConfig, release_directory_root:
     return asset_config.full_name in repo.tags
 
 
+
+def validate_new_release(asset_config: assets.AssetConfig, release_directory_root: Path) -> str:

```

Comment:
Please rename to something that indicates what the function is doing now. Something like `get_latest_release_tag_version`?